service/src/main/java/uk/gov/hmcts/reform/fpl/model/migration/MigratedRespondent.java

```diff
@@ -0,0 +1,16 @@
+package uk.gov.hmcts.reform.fpl.model.migration;
+
+import com.fasterxml.jackson.annotation.JsonProperty;
+import lombok.AllArgsConstructor;
+import lombok.Builder;
+import lombok.Data;
+import uk.gov.hmcts.reform.fpl.model.PartyExtended;
+
+@Data
+@Builder
+@AllArgsConstructor
+public class MigratedRespondent {
+    @JsonProperty("party")
```

Comment:
I bet it is not needed since field has the same name. Please try to remove it.

Reply:
I think this is left over from a previous name, good spot 👍